scout/server/blueprints/variants/views.py

```diff
@@ -80,6 +94,14 @@ def variants(institute_id, case_name):
 
     form.gene_panels.choices = panel_choices
 
+    # update status of case if vistited for the first time
```

Comment:
visited

Reply:
https://twitter.com/wistikent/status/660560676595847168

Reply:
Sorry, didn't know it was a citation! :D

Reply:
😸 Freshly, and post-hoc, invented verb!